_includes/_blogcontent/2024-03-06-introducing-scix-ambassador-program.md

```diff
@@ -0,0 +1,37 @@
+### Introducing the NASA Science Explorer Ambassador Program
+
+The [NASA Science Explorer (SciX)](https://scixplorer.org) team is thrilled to announce the launch of the SciX Ambassador Program! This initiative marks a new chapter in the evolution of the NASA Astrophysics Data System (ADS) into NASA SciX, with a significant leap in promoting interdisciplinary communication and research. To further this mission, we are seeking approximately 12 researchers to become our Lead Ambassadors, representing diverse disciplines across NASA, including Astrophysics, Planetary Science, Earth Science, Heliophysics, and Biological & Physical Sciences. Our goal is to drive forward the vision of interdisciplinary engagement and innovation as a cornerstone of NASA SciX.
+
+<div class="text-center">
+    <img class="img-thumbnail" src="{{ site.baseurl }}/blog/images/blog_2023-03-06-SciX_AmbassadorProgram_Flyer.png" />
+<br>
+<em>SciX Ambassador Program Flyer
+</em>
+</div>
+<br>
+
+The SciX Lead Ambassadors will have the crucial task of educating their communities about the benefits of SciX, and they will do so by presenting on the platform at least twice a year in various formats. To learn more about SciX and gain community engagement skills to better educate their communities the ambassadors will participate in a 2.5-day workshop at the [Center for Astrophysics | Harvard & Smithsonian](https://www.cfa.harvard.edu/) from August 21 to 23, 2024. This workshop will provide a unique opportunity for the ambassadors to connect directly with the NASA SciX team, learn essential community engagement skills, and receive in-depth training on the NASA SciX platform. This experience will equip them with the tools and knowledge needed to effectively promote SciX within their communities.
```

Comment:
Markdown interprets the pipe symbol (`|`) as a special character for making a table - you have to escape it (`\|`) to actually display a pipe. This will render properly: 
`Center for Astrophysics \| Harvard & Smithsonian`